src/Marten.Storyteller/Fixtures/EventStore/InlineAggregationFixture.cs

```diff
@@ -74,7 +75,7 @@ public IGrammar QuestPartyShouldBe()
         }
 
         [FormatAs("Members should be {Members}")]
-        public string[] Members()
+        public List<string> Members()
```

Comment:
Why did you have to do this? This wouldn't be serialized anywhere.

Reply:
This was done to align other changes, but it's reverted.